ui/component/markdownLink/view.jsx

```diff
@@ -46,7 +53,10 @@ function MarkdownLink(props: Props) {
   const protocol = href ? protocolRegex.exec(href) : null;
   const isMention = href && href.startsWith('lbry://@');
   const mentionedMyChannel =
-    isMention && (myChannelUrls ? myChannelUrls.some((url) => url.replace('#', ':') === href) : false);
+    isMention &&
+    activeChannelClaim &&
+    activeChannelClaim.canonical_url &&
+    activeChannelClaim.canonical_url.replace('#', ':') === href;
 
   React.useEffect(() => {
     if (mentionedMyChannel && setUserMention) setUserMention(true);
```

Comment:
I see `mentionedMyChannel` is now specifically the active-channel instead of any-of-my-channel.  

Ok, but the `setUserMention(true)` line only sets it to true, so when you change the active channel to something else, the mentioned comment stays highlighted.  Mix of old and new behavior.  

Probably need to always set `setUserMention(true|false)`

Reply:
Yeah, I've made a few adjustments because you can no longer change your channel in the chat. I mean, you can change it, but you won't be able to write anything. Therefore a scenario in which a message from another one of your channels gets highlighted should no longer be possible.